Kernel/Process.cpp

```diff
@@ -4045,31 +4045,26 @@ int Process::sys$mount(const Syscall::SC_mount_params* user_params)
     auto& target_custody = custody_or_error.value();
 
     RefPtr<FS> fs;
+    auto fd = file_description(source_fd);
+    if (fd.is_null())
+        return -EFAULT;
```

Comment:
`EBADF`

Reply:
or maybe we should have `EBADFD`? I don't see it in errno_numbers.h, but I do see it in other file in my system.